### DISCUSSION TITLE
freerdp 3.4.0-5 on Arch Linux just doesnt work

### DISCUSSION THREAD

**Discussion by emiliskvg** [FreeRDP/FreeRDP#10018] [Q&A] [answered]
Hello, maybe it's just an Arch Linux issue, but after the recent update "2:3.4.0-5", trying to run any command just gives me "command not found: xfreerdp".

Downgrading to 3.3 or easlier works just fine. So I'm wondering what changed or was broken in the 3.4 version. I tried it with zsh and bash, and with two different systems and a clean install too. The new version just doesnt work...

**Comment by akallabeth** [comment, marked as answer]
that is definitely an arch issue.
check if build with `-DWITH_BINARY_VERSIONING=ON` then the commands changed (`xfreerdp -> xfreerdp3` et al)

**Reply by emiliskvg** [reply]
Yes, the new command is xfreerdp3. :)
This solved my issue.

**Reply by pnowack** [reply]
> then the commands changed (xfreerdp -> xfreerdp3 et al)

I think this was done, because Arch provides both FreeRDP3 (in the `freerdp` pkg) and FreeRDP2 (now as `freerdp2`) and both pkgs provide the CLI binaries, so at least one needs to be versioned. Better would be if the binaries in the `freerdp2` would be versioned.

**Reply by rogue-agent** [reply]
This is so dumb.

**Reply by akallabeth** [reply]
@rogue-agent wrong target audience, complain in arch tracker/forum

**Reply by 1010dvpt** [reply]
Same issue on Solus
